doc/SPIRE101.md

```diff
@@ -0,0 +1,96 @@
+
+
+# Overview
+
+Follow these walkthrough to set up your docker development environment and expose the Workload API. A workload client cmdline simulates a workload fetching SVIDs for its registered SPIFFE IDs.
```

Comment:
This walkthrough will guide you through the steps needed to setup a running example of a SPIRE server and agent.  Interaction with the Workload API [Link] will be simulated via a command line tool.